src/model/transformations/collected-variable.js

```diff
@@ -3,13 +3,24 @@ import {
   VARIABLES_TYPES,
   DATATYPE_TYPE_FROM_NAME, DATATYPE_NAME
 } from 'constants/pogues-constants';
+import { element } from 'prop-types';
 const { COLLECTED } = VARIABLES_TYPES;
 
 export function remoteToStore(
   remote = [],
   responsesByVariable,
   codesListsStore,
+  variableclarification,
 ) {
+  remote.map(variable => {
```

Comment:
a .map function should return something. 
If it is not the case, you should prefer .forEach